config/runtimes/kserve-ovms.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-ovms
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8888'
+    prometheus.kserve.io/path: "/metrics"
+  supportedModelFormats:
+    - name: openvino_ir
+      version: opset11
+      autoSelect: true
+    - name: onnx
+      version: "1"
+    - name: tensorflow
+      version: "1"
+      autoSelect: true
```

Comment:
@sivanantha321 I think we will need to set the priority here right?

Reply:
yeah, if auto selection is enabled, then it is better to provide priority

Reply:
set `priority:1` option